src/helpers/articles.js

```diff
@@ -8,9 +8,9 @@ const { Article } = Models;
  */
 class ArticleModel {
   /**
-   * @description - This method is responsible for quering the database for an article
-   * @static
-   * @returns {object} - object representing response message
+   *
```

Comment:
@omokaro-faith, I removed the description for this method (`createArticle`), because it was misleading. What I, however, failed to do was write in an appropriate one - which I'd get done within the hour.